getting-started-tutorial/01-mlrun-basics.ipynb

```diff
@@ -248,38 +260,47 @@
        "'/v3io/projects/{{run.project}}/artifacts'"
       ]
      },
-     "execution_count": 3,
+     "execution_count": 5,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "mlconf.artifact_path"
+    "print(f'Artifacts path: {artifact_path}')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Working from remote \n",
-    "This tutorial showcases how to run your project in this built-in jupyter service, however, as a developer you may want to develop from <br>\n",
-    "remote using your own IDE (local jupyter, pycharm etc..). <br>\n",
-    "Go to this link to learn how to work with a remote IDE <br>\n",
-    "https://mlrun.readthedocs.io/en/latest/remote.html"
+    "<a id=\"gs-tutorial-1-setup-remote-env\"></a>"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Setting your project"
+    "### Using MLRun Remotely\n",
+    "\n",
+    "This tutorial is aimed at running your project from a local Jupyter Notebook service in the same environment in which MLRun is installed and running.\n",
+    "However, as a developer you might want to develop your project from a remote location using your own IDE (such as a local Jupyter Notebook or PyCharm), and connect to the MLRun environment remotely.\n",
+    "To learn how to use MLRun from a remote IDE, see the [MLRun documentation](https://mlrun.readthedocs.io/en/release-v0.6.x-latest/remote.html)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Projects in the platform are used to package multiple functions, workflows, and artifacts.\n",
+    "<a id=\"gs-tutorial-1-project-setup\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting Up Your Project\n",
+    "\n",
+    "MLRun and platform projects are used to package multiple functions, workflows, and artifacts.\n",
     "Projects are created by using the `new_project` MLRun method, which receives the following parameters:\n",
```

Comment:
@Sharon-iguazio @gilad-shaham need to remove all the new/load project stuff from NB 1-3 (doc + code), it is not used correctly

Reply:
@yaronha this isn't my call. It was discussed at length between you and @gilad-shaham  on Thursday and Gilad told me to keep the project code and doc because he says we rely on the new-project code in other parts of the tutorial. => Please close this amongst yourself and give me an agreed bottom line as to the changes to make.